test/unit/pricing/subscription/subscription.test.js

```diff
@@ -15,10 +15,11 @@ describe('Recurly.Pricing.Subscription', function () {
           country: 'US',
           postal_code: 'NoTax'
         })
-        .done(function (price) {
+        .done((price) => {
           assert.equal(price.taxes.length, 0);
           assert.equal(price.now.tax, '0.00');
           assert.equal(price.next.tax, '0.00');
+          assert.equal(this.pricing.items.plan.price.USD.symbol, '$');
```

Comment:
This test is specifically targeting taxation concerns. I think testing the internal state of the place price here is irrelevant.